buildscripts/parquet-reader-conda-recipe/meta.yaml

```diff
@@ -14,11 +14,11 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake >=3.2
     - python 3.6.*
-    - pyarrow ==0.14.1
-    - arrow-cpp ==0.14.1
+    - pyarrow ==0.15.0
```

Comment:
Is it possible to have this version of the required package in one place for whole project? It might be some macro variable or something like this.

Reply:
Example with eliminating duplication inside one file https://github.com/IntelPython/hpat/pull/252.
I think use one version for whole project is not possible because all places plays different roles. Some places points range of available versions for use in customer environment, some points to concrete version we use for building and development.

@Vyacheslav-Smirnov have you some ideas about it?

Reply:
This [article](https://caremad.io/posts/2013/07/setup-vs-requirement/) explains differences between versions in Python infrastructure files.